### PR TITLE
Cypress/Playwright/TestCafe - Update available versions

### PR DESCRIPTION
**Pull request by FriggaHel**
### Description

Updates the available versions as we are upgrading the frameworks.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/web-apps/automated-testing/cypress.md [hunk 1]
```diff
@@ -66,6 +66,17 @@ values={[
       <th>Supported Browsers</th>
       <th>End of Life</th>
     </tr>
+    <tbody>
+      <tr>
+        <td rowspan='2'>12.6.0</td>
+        <td><b>macOS:</b> 11.00, 12</td>
+        <td rowspan='2'>Chrome, Firefox, Microsoft Edge, Webkit (Experimental)</td>
+        <td rowspan='2'>Mar 01, 2024</td>
```
Comment (aliciakrebs) [inline docs/web-apps/automated-testing/cypress.md:74]:
```suggestion
        <td rowspan='2'>March 1, 2024</td>
```

File: docs/web-apps/automated-testing/playwright.md [hunk 2]
```diff
@@ -68,6 +68,17 @@ values={[
       <th>Supported Browsers</th>
       <th>End of Life</th>
     </tr>
+    <tbody>
+      <tr>
+        <td rowspan='2'>1.31.1</td>
+        <td><b>macOS:</b> 11.00, 12</td>
+        <td rowspan='2'>Chromium, Chrome, Firefox, Webkit</td>
+        <td rowspan='2'>Mar 01, 2024</td>
```
Comment (aliciakrebs) [inline docs/web-apps/automated-testing/playwright.md:76]:
```suggestion
        <td rowspan='2'>March 1, 2024</td>
```

File: docs/web-apps/automated-testing/testcafe.md [hunk 3]
```diff
@@ -67,6 +67,22 @@ values={[
       <th>Supported Browsers</th>
       <th>End of Life</th>
     </tr>
+    <tbody>
+      <tr>
+        <td rowspan='3'>2.3.1</td>
+        <td><b>macOS:</b> 11.00, 12</td>
+        <td>Safari, Chrome, Firefox, Microsoft Edge</td>
+        <td rowspan='3'>Mar 01, 2024</td>
```
Comment (aliciakrebs) [inline docs/web-apps/automated-testing/testcafe.md:75]:
```suggestion
        <td rowspan='3'>March 1, 2024</td>
```

**Review by aliciakrebs — APPROVED** [review]

File: docs/web-apps/automated-testing/cypress.md (hunk 1 above)
Comment (FriggaHel) [reply docs/web-apps/automated-testing/cypress.md:74]: Shouldn't we keep those values consistent with the others ?

Comment (aliciakrebs) [reply docs/web-apps/automated-testing/cypress.md:74]: ah, yes - looks like we have not been consistent in not following ap style for abbreviating  months.

**Review by aliciakrebs — APPROVED** [review]

**Pull request merged**